### PR TITLE
Add metadata to create .cab files for the LVFS

### PR DESCRIPTION
**Pull request by hughsie**
This will create files of the format Logitech-Unifying-RQR024.005_B00029.cab

Signed-off-by: Richard Hughes <richard@hughsie.com>

### CODE REVIEW

File: RQR12/RQR12.05/com.logitech.Unifying.RQR12.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <!-- USB\VID_046D&PID_AAAA -->
+    <firmware type="flashed">9d131a0c-a606-580f-8eda-80587250b8d6</firmware>
+  </provides>
+  <url type="homepage">http://support.logitech.com/en-us/software/unifying</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Proprietary</project_license>
+  <developer_name>Logitech</developer_name>
+  <releases>
+    <release urgency="high" version="RQR12.07_B0029" date="2017-05-02">
+      <checksum filename="RQR12.07_B0029.hex" target="content"/>
```
Comment (ogay) [inline RQR12/RQR12.05/com.logitech.Unifying.RQR12.metainfo.xml]: RQR12.05_B0028 ?

Comment (hughsie) [reply RQR12/RQR12.05/com.logitech.Unifying.RQR12.metainfo.xml]: Good catch, will respin now.

**Comment by ogay** [comment]
We discussed it and we would prefer if every Makefile and xml file comes in its own folder, called 'lvfs', e.g., 

RQR12/RQR12.07/RQR12.07_B0029.hex
RQR12/RQR12.07/RQR12.07_B0029.txt
RQR12/RQR12.07/lvfs/com.logitech.Unifying.RQR12.metainfo.xml
RQR12/RQR12.07/lvfs/Makefile

and the Makefile would create the .cab in the same level as the level of the Makefile: i.e., 
RQR12/RQR12.07/lvfs/Logitech-Unifying-RQR12.07_B0029.cab

**Comment by hughsie** [comment]
As you wish; one sec.

File: RQR24/RQR24.03/lvfs/Makefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2017 Richard Hughes <richard@hughsie.com>
+
+VENDOR=Logitech
+PROJECT_NAME=Unifying
+VERSION=RQR024.003_B00027
```
Comment (ogay) [inline RQR24/RQR24.03/lvfs/Makefile]: VERSION=RQR24.03_B0027
(there were three extra 0s)

File: RQR24/RQR24.05/lvfs/Makefile [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2017 Richard Hughes <richard@hughsie.com>
+
+VENDOR=Logitech
+PROJECT_NAME=Unifying
+VERSION=RQR024.005_B00029
```
Comment (ogay) [inline RQR24/RQR24.05/lvfs/Makefile]: VERSION=RQR24.05_B0029
(there were three extra 0s)

**Review by ogay — CHANGES_REQUESTED** [review]
Sorry I didn't catch this during the last review.

The cab filenames have an inconsistent format; the RQR12 are the correct one.

It also appears in the commit message:
Logitech-Unifying-RQR024.005_B00029.cab should be Logitech-Unifying-RQR24.05_B0029.cab (three extra 0s)

**Comment by hughsie** [comment]
My error entirely, sorry. New version pushed.

**Pull request merged**